packages/ui-view/src/View/__examples__/View.examples.ts

```diff
@@ -83,6 +83,18 @@ export default {
           props.textAlign !== 'center')) ||
       // Only generate a 1 variation for non-'0' borderWidth
       (props.borderWidth !== '0' &&
+        (props.background !== 'transparent' ||
+          props.borderRadius !== '0' ||
+          props.position !== 'static' ||
+          props.shadow !== 'none' ||
+          props.textAlign !== 'center')) ||
+      (props.overscrollBehavior === 'none' &&
+        (props.background !== 'transparent' ||
+          props.borderRadius !== '0' ||
+          props.position !== 'static' ||
+          props.shadow !== 'none' ||
+          props.textAlign !== 'center')) ||
+      (props.overscrollBehavior === 'auto' &&
```

Comment:
something is wrong with this, I dont see any new examples in Chromatic

Reply:
I already accepted them.